docs/source/kale.prepdata.rst

```diff
@@ -31,7 +31,7 @@ kale.prepdata.tensor\_reshape module
    :undoc-members:
    :show-inheritance:
 
-kale.prepdata.chem\_transform module
+kale.prepdata.chem_transform module
```

Comment:
Does this change work on doc build? As I remeber it failed without "\" on my local build.

Reply:
It seems work both ways on rtd. I made the changes trying to find out why docs are missing on rtd